### ISSUE TITLE
Status and Roadmap

### ISSUE THREAD

**Issue by antont** [FritzsHero/RoadArchitect#2]
Hi - very cool to see that you have worked on this after the original devs seem to have gone elsewhere.

Are you still planning to continue, or is this abandoned now again?

We are not using RoadArchitect yet but tested it quite a lot already a year ago, now again making offers to potential customers where this might be great. Nothing certain yet.

**Comment by embeddedt** [comment]
@FritzsHero is still working on it; it's just that changes are happening on other branches, not the `master` one which everyone sees by default. The [`RewrittenAPI`](https://github.com/FritzsHero/RoadArchitect/tree/RewrittenAPI) branch was last updated 2 days ago.

**Comment by antont** [comment]
Oops forgot to check branches here, only looked at them in the original MicroGSD repo - sorry for the noise, and thanks for info!

**Comment by FritzsHero** [comment]
Yes, I still have plans on updating this until we have support for roundabouts and maybe even more intersections.
I also still need to write a Wiki, Update the Graphics and Improve comments on the scripts.

Since I do not have much time after work for this, it might be a slow development.
Please feel free to discuss ideas or problems about this project.


Best Regards
Fritzs

**Comment by 1steelbird** [comment]
@FritzsHero  Hello first of all thank you to keep this updated and working.

Have you been working on this ? Im having issues making my own roundabout , with this working will be awesome, then all its working properly.

best regards :)

**Comment by jperezCurso** [comment]
@FritzsHero Thank you very much for continuing this project. 
I'm learning Unity and I was using the original Road Architect and just saw this fork.
Looking for a tool to make roads, this seems to me to be one of the best.
I have a question:
     Is it possible to access the spline of the road to know in which direction the cars have to go?
Thanks and regards.
